### PR TITLE
feat: onboard Teams send proactive message sample to toolkit

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
ADO:
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/13999494

Screenshots:
image
image
image

E2E TEST: NA

### CODE REVIEW

File: packages/fx-core/src/common/samples-config.json [hunk 1]
```diff
@@ -136,6 +136,20 @@
             "time": "5min to run",
             "configuration": "Ready for local debug",
             "suggested": false
+        },
+        {
+            "id": "bot-proactive-messaging-teamsfx",
+            "title": "Proactive Messaging",
+            "shortDescription": "This is a sample which shows how to save user's conversation reference information to send proactive reminder message using Bots.",
+            "fullDescription": "This is a sample which shows how to save user's conversation reference information to send proactive reminder message using Bots.",
```
Comment (tecton) [inline packages/fx-core/src/common/samples-config.json:144]: same with short one?

File: packages/vscode-extension/src/controls/SampleGallery.tsx [hunk 2]
```diff
@@ -80,15 +81,14 @@ export default class SampleGallery extends React.Component<any, any> {
...
-                baseUrl={this.state.baseUrl}
                 highlightSample={this.highlightSample}
               />
             </div>
           </div>
         )}
         {this.state.highlightSample != "" && (
           <SampleDetailPage
-            baseUrl={this.state.baseUrl}
+            baseUrl={hightSample.url}
```
Comment (tecton) [inline packages/vscode-extension/src/controls/SampleGallery.tsx:91]: Does this mean highlightSample? it's a bit confusing.

**Review by tecton — DISMISSED** [review]

File: packages/fx-core/src/common/samples-config.json (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/src/common/samples-config.json:144]: Updated

File: packages/vscode-extension/src/controls/SampleGallery.tsx (hunk 2 above)
Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/src/controls/SampleGallery.tsx]: updated

**Review by tecton — APPROVED** [review]

**Pull request merged**